src/routers/webhooks.py

```
@@ -0,0 +1,91 @@
+from src.webhooks.events import QueueEmptyPayload
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix="/webhooks",
+    tags=["webhooks"],
+)
+
+
+@router.post("", response_model=schemas.WebhookEndpoint)
+async def get_or_create_webhook_endpoint(
```

Ruff (0.12.2)

Missing return type annotation for public function `get_or_create_webhook_endpoint`

(ANN201)